recipes/mgrs/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mgrs" %}
+{% set version = "1.3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: b8e4eaab7a4057910dd18fce9047ab2487a9d43a5c205510465709f62156d02c
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
+  run:
+    - python
+    - setuptools
```

Comment:
I think you do not need ``setuptools`` and ``wheel`` to run this project, don't you?

Reply:
:/
It seems to be necessary for windows then, that is ok